app/assets/stylesheets/custom.css.scss

```diff
@@ -1,9 +1,28 @@
+$light-gray: #f7f7f7;
+
 body {
   padding-top: 50px;
 }
 
 .bg-light-gray {
-    background-color: #f7f7f7;
+  background-color: $light-gray;
+}
+
+
+.page-header {
+  text-align: center;
+}
+
+.thumbnail {
+  background-color: $light-gray !important;
+}
+
+.thumbnail-label {
+  text-align: center;
+}
+
+.progress {
+  background-color: white !important;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used